apps/zui/src/electron/run-main/before-boot.ts

```diff
@@ -5,12 +5,16 @@ import {windowsPre25Exists} from "../windows-pre-25"
 import {MainArgs} from "./args"
 import {setLogLevel} from "../set-log-level"
 import {runMigrations} from "./run-migrations"
+import log from "electron-log"
 
 app.disableHardwareAcceleration()
 
 export async function beforeBoot(
   args: Partial<MainArgs>
 ): Promise<string | null> {
+  // Ensure all console logs go through electron-log
+  console.log = log.log
+
```

Comment:
This one caused the `console.log()` messages I'd added in `packages/zed-node/src/lake.ts` to show in the `packages/zui-player/run/playwright-itest/Pool Loads (succeeses)/1/logs/main.log`.